meumobi/AMS.Connect
# Build/update dependencies

# How to Test
1. Create .env file based on https://dashboard.heroku.com/apps/limitless-sea-32755/settings config vars
FIREBASE_CONFIG_PATH=ams-report-test-firebase-adminsdk-z0qg1-c777197914.json to test on int
2. Run Docker VM
$ docker-compose up -d

3. Run command
$ docker exec amsconnect_ams.connect_1 php artisan providers:perform --mode=publish

4. Access Reports
https://console.firebase.google.com/u/1/project/ams-report-test/database/ams-report-test/data/reports 

5. Check Adsense.inventarie
 https://console.firebase.google.com/u/1/project/ams-report-test/database/ams-report-test/data/reports/2019-10-27_handnews_adsense/raws/2019-10-27_1339484712
Adsense.inventarie = AMS Market Place
```js
{
  "annonceur" : "NA",
  "campagne" : "NA",
  "date" : "2019-10-27",
  "emplacement" : "Handnews_horizontal_top_header_desktop",
  "format" : "600 x 100",
  "impressions envoyees" : "Unknown",
  "impressions facturables" : "NA",
  "impressions prises" : "0",
  "impressions reçues" : "4",
  "inventaire" : "AMS Market Place",
  "key" : "1339484712",
  "marge" : "Unknown",
  "partenaire" : "Adsense",
  "position" : "desktop - top",
  "revenu" : "0.0000000000",
  "revenu net" : "Unknown",
  "site" : "Handnews",
  "uid" : "2019-10-27_1339484712"
}
```

6. Check Criteowf
There is no criteowf for 2019-10-27 date just before

7. All providers were performed
